lib/core/test/unit/Cardano/Wallet/Primitive/TypesSpec.hs

```diff
@@ -302,6 +323,63 @@ prop_2_6_2 (ins, u) =
             ===
         balance u - balance (u `restrictedBy` ins)
 
+{-------------------------------------------------------------------------------
+                        UTxO statistics Properties
+-------------------------------------------------------------------------------}
+
+-- | The 'total' stake in the statistics is the UTxO's balance
```

Comment:
So just that the "total" (API terminology) field is the "sum" of the UTxO, right.

Is "stake" the correct term here?

Reply:
I think it is :)

Reply:
What if the UTxO contains funds within enterprise addresses (i.e which are non-stakeable)?

![IMG_EB54401351E6-1](https://user-images.githubusercontent.com/304423/61765924-52cd2880-addf-11e9-84c8-1ab2f07b751c.jpeg)

(https://hydra.iohk.io/build/790053/download/1/delegation_design_spec.pdf)